docs/src/operations/setup-a-validator.md

```diff
@@ -508,10 +510,11 @@ through the validator log output.
 
 ### Solana Validators
 
-After you have verified that your validator is in gossip, you can verify that
-your validator has joined the network using the `solana validators` command. The
-command lists all validators in the network, but like before, we can `grep` the
-output for the validator we care about:
+After you have verified that your validator is in gossip, you need to stake some
+SOL to your validator. Then you can verify that your validator has joined the
+network using the `solana validators` command. The command lists all validators
+in the network, but like before, we can `grep` the output for the validator we
+care about:
```

Comment:
This is a good addition, but I think it could be even more helpful to mention that the stake must be active (this has been a somewhat frequent question in Discord):

```suggestion
After you have verified that your validator is in gossip, you should stake some
SOL to your validator. Once the stake has activated (which happens at the start
of the next epoch), you can verify that your validator is ready to be a voting
participant of the network with the `solana validators` command. The command
lists all validators in the network, but like before, we can `grep` the output
for the validator we care about:
```